filechanges/etc/fstab

```diff
@@ -1,5 +0,0 @@
-proc            /proc           proc    defaults          0       0
-/dev/mmcblk0p1  /boot           vfat    defaults,user,rw,umask=000          0       2
-/dev/mmcblk0p2  /               ext4    defaults,noatime  0       1
-# a swapfile is not a swap partition, so no using swapon|off from here on, use  dphys-swapfile swap[on|off]  for that
-/dev/mmcblk0p3  none            swap    sw                0       0
```

Comment:
Hmm I think we actually need to test the implication of `data=writeback` too. Previously we've had quite bad performance issues relating to Spotify data caching, which creates a bunch of files as it goes. Just need to make sure this option doesn't make that any worse.